### PR TITLE
GitHub: Added for review the git command usage

### PR DESCRIPTION
**Pull request by david1412**
Please leave your comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# testpr
\ No newline at end of file
+# Github basics
```
Comment (manisharayudu) [inline README.md:1]: Please adjust indentation.

**Review by manisharayudu — COMMENTED** [review]
I left few comments.

File: README.md (hunk 1 above)
Comment (david1412) [reply README.md:1]: ok done

**Review by manisharayudu — APPROVED** [review]
Approved

**Pull request merged**